### PR TITLE
vim-patch:86b4816766d9

### PR DESCRIPTION
**Pull request by clason**
Update runtime files

https://github.com/vim/vim/commit/86b4816766d976a7ecd4403eca1f8bf6b4105800

@zeertzjq is the change to `doc/windows.txt` applicable?

Co-authored-by: Bram Moolenaar <Bram@vim.org>

### CODE REVIEW

**Comment by zeertzjq** [comment]
> is the change to doc/windows.txt applicable?

Yes

File: runtime/doc/visual.txt [hunk 1]
```diff
@@ -139,6 +139,11 @@ gN			Like |gn| but searches backward, like with `N`.
 			environment variable or the -display argument).  Only
 			when 'mouse' option contains 'n' or 'a'.
 
+<LeftMouseNM>		Internal mouse code, used for clicking on the status
+<LeftReleaseNM>		line to focus a window.  NM stands for non-mappable.
+			You cannot use these, but they might show up in some
+			places.
```
[the remark this answers is not in the document]
Comment (clason) [reply runtime/doc/visual.txt:145]: So should I remove it or leave it in?

[the remark this answers is not in the document]
Comment (seandewar) [reply runtime/doc/visual.txt:145]: I'd remove it imo

**Review by zeertzjq — APPROVED** [review]

**Review by seandewar — APPROVED** [review]
LGTM. Shame Zig is missing its autoload file, but that's on Bram. :P

**Comment by clason** [comment]
I think there's gonna be a supplementary runtime file update soon (today?), so might make sense to wait for and include that?

**Comment by seandewar** [comment]
Waiting sounds fine. I've alerted Bram about the missing file. :pray:

**Comment by seandewar** [comment]
https://github.com/vim/vim/commit/86b4816766d976a7ecd4403eca1f8bf6b4105800#r92400252
https://github.com/vim/vim/commit/84dbf855fb2d883481f74ad0ccf3df3f8837e6bf
No idea when that'll be :innocent:

**Comment by clason** [comment]
Yeah... Maybe you should have mentioned that the zig ftplugin is completely broken without it?

**Comment by seandewar** [comment]
Good point. I thought it may have been clear from the BufWritePre autocmd that I attached the comment to, but I've mentioned it explicitly now just in case. :P

**Review by seandewar — APPROVED** [review]
:)

**Comment by Cethrivias** [comment]
Hey guys. This is an old issue, but maybe you can explain to me what triggers `function! zig#fmt#Format() abort` and how can I disable it? I don't want it to be called ever on my local machine. Is there a way to exclude the file from autoloading? Or something of the sorts

**Comment by seandewar** [comment]
@Cethrivias It was triggered by a BufWritePre autocommand in `runtime/ftplugin/zig.vim`. Use `let g:zig_fmt_autosave = 0` to disable it.

The auto-formatting has since been removed in #28904, so this workaround won't be needed after 0.11.

**Comment by Cethrivias** [comment]
Thank you!

**Pull request merged**